### PR TITLE
Archive

### PR DESCRIPTION
**Pull request by vermasavita**

### CODE REVIEW

**Review by Jay-Gadhiya — APPROVED** [review]
- code looks clean
- indentation is good
-api calls are correct

File: src/pages/archive-page/ArchiveNote.jsx [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+                <ArchiveNoteContainer
+                  key={archiveNote._id}
+                  archiveNote={archiveNote}
+                  archiveNoteId={archiveNote._id}
+                  archiveNoteTitle={archiveNote.title}
+                  archiveNoteContent={archiveNote.noteContent}
+                  callDeleteFromArchivesHandler={callDeleteFromArchivesHandler}
+                  callRestoreFromArchivesHandler={
+                    callRestoreFromArchivesHandler
+                  }
```
Comment (Jay-Gadhiya) [inline src/pages/archive-page/ArchiveNote.jsx:53]: - you can pass item and destructure it in `ArchiveNoteContainer`

**Review by Jay-Gadhiya — APPROVED** [review]

File: src/pages/archive-page/ArchiveNote.jsx [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+} from "../../service";
+
+const ArchivedNote = () => {
+  const { noteState, noteDispatch } = useNote();
+  const { authState } = useAuth();
+  const { token } = authState;
+  const { archiveNoteState, archiveNoteDispatch } = useArchive();
+  const { archivedNote } = archiveNoteState;
+  console.log(archivedNote);
+  console.log(noteState.notes);
```
Comment (Naman-Saxena1) [inline src/pages/archive-page/ArchiveNote.jsx:18]: Can remove consoles when merging PR

**Review by Naman-Saxena1 — APPROVED** [review]
Code looks fine and functionality is also working fine with test creds.
I suggest you improve PR description by mentioning:
1. All new features in detail
2. Files to review and ignore
3. Screenshots of new features
4. Any extra details that reviewer should know

You can refer PR description of my old PR for context
https://github.com/Naman-Saxena1/Socioztron-Social_Media/pull/2

File: src/pages/archive-page/ArchiveNote.jsx (hunk 2 above)
Comment (vermasavita) [reply src/pages/archive-page/ArchiveNote.jsx]: Removed

**Comment by vermasavita** [comment]
> Code looks fine and functionality is also working fine with test creds. I suggest you improve PR description by mentioning:
> 
> 1. All new features in detail
> 2. Files to review and ignore
> 3. Screenshots of new features
> 4. Any extra details that reviewer should know
> 
> You can refer PR description of my old PR for context [Naman-Saxena1/Socioztron-Social_Media#2](https://github.com/Naman-Saxena1/Socioztron-Social_Media/pull/2)

Sure, Naman, I'll keep this in mind from the next time onwards. Thanks :-)

**Pull request merged**